vmaas/version.go

```diff
@@ -0,0 +1,22 @@
+package vmaas
+
+import (
+	"os"
+)
+
+func vmaasVersion(opts *options) *string {
+	path := opts.vmaasVersionFilePath
+	if path == "" {
+		return nil
+	}
+
+	dat, err := os.ReadFile(path)
+	if err != nil {
+		return nil
+	}
+
+	skipLabel := min(len(dat), 8)
+	skipNewline := max(0, len(dat)-1)
+	res := string(dat[skipLabel:skipNewline])
```

Comment:
what is the file we're planning on reading, `constants.py`?
Could we introduce a similar approach as in Patch with [VERSION file](https://github.com/RedHatInsights/patchman-engine/blob/master/VERSION)?  In Patch case it is created by a [github action](https://github.com/RedHatInsights/patchman-engine/blob/master/.github/workflows/semantic-release.yml#L19C9-L24) but I think it should be able to configure [python-semantic-release](https://github.com/RedHatInsights/vmaas/blob/master/.github/workflows/semantic-release.yml) or anything else currently writing to `constants.py` to create `VERSION` file just with the app version so we don't need to skip labels, lines etc.

I'd assume that constants.py will be dropped once we have a webapp go rewrite